microcloud/service/lxd.go

```diff
@@ -400,7 +400,7 @@ func (s *LXDService) Restart(timeoutSeconds int) error {
 	}
 
 	_, _, err = c.RawQuery("PUT", "/internal/shutdown", nil, "")
-	if err != nil {
+	if err != nil && err.Error() != "Shutdown already in progress" {
```

Comment:
What http code do you get with this error? We should perhaps change this to something you can detect programmatically that wont break if the message is changed. (This is the sort of thing juju used to do and it didn't go well).

Reply:
It's just the default `500 Internal Server Error`. I can make it `429 Too Many Requests` and check for that instead.

Reply:
Yeah thats a good idea, thanks!